packages/react/src/utils.ts

```diff
@@ -31,3 +32,13 @@ export function publicationId(id: string): PublicationId {
   // for now just asserts the type, in future it will enforce a format
   return id as PublicationId;
 }
+
+/**
+ * Ensures that the given value is a valid ImageSizeTransform
+ *
+ * @group Helpers
+ */
+export function imageSizeTransform(value: string): ImageSizeTransform {
+  // for now just asserts the type, in future it will enforce a format
+  return value as ImageSizeTransform;
+}
```

Comment:
With the template literal type... you won't need this to be a branded type anymore so the need for this helper kinda disappear.